specification/search/data-plane/Azure.Search/preview/2024-11-01-preview/searchindex.json

```diff
@@ -0,0 +1,3220 @@
+              },
+              {
+                "value": "delete",
+                "name": "Delete",
+                "description": "Removes the specified document from the index. Any field you specify in a delete operation other than the key field will be ignored. If you want to remove an individual field from a document, use merge instead and set the field explicitly to null."
+              }
+            ]
+          },
+          "x-ms-client-name": "ActionType",
+          "x-nullable": false,
+          "description": "The operation to perform on a document in an indexing batch."
+        }
+      },
+      "additionalProperties": true,
+      "description": "Represents an index action that operates on a document."
+    },
+    "IndexingResult": {
+      "properties": {
+        "key": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The key of a document that was in the indexing request."
+        },
+        "errorMessage": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The error message explaining why the indexing operation failed for the document identified by the key; null if indexing succeeded."
+        },
+        "status": {
+          "x-ms-client-name": "Succeeded",
+          "type": "boolean",
+          "x-nullable": false,
+          "readOnly": true,
+          "description": "A value indicating whether the indexing operation succeeded for the document identified by the key."
+        },
+        "statusCode": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": false,
+          "readOnly": true,
+          "description": "The status code of the indexing operation. Possible values include: 200 for a successful update or delete, 201 for successful document creation, 400 for a malformed input document, 404 for document not found, 409 for a version conflict, 422 when the index is temporarily unavailable, or 503 for when the service is too busy."
+        }
+      },
+      "required": [
+        "key",
+        "status",
+        "statusCode"
+      ],
+      "description": "Status of an indexing operation for a single document."
+    },
+    "IndexDocumentsResult": {
+      "properties": {
+        "value": {
+          "x-ms-client-name": "Results",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/IndexingResult"
+          },
+          "description": "The list of status information for each document in the indexing request."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "Response containing the status of operations for all documents in the indexing request."
+    },
+    "QueryDebugMode": {
+      "type": "string",
+      "enum": [
+        "disabled",
```

Comment:
Appears that `all` is also an allowed value, should that be added here? And a question on usage of this, is this similar to `RegexFlags` where multiple values can be passed? Ex, `semantic,vector`?